srcpkgs/libxml2-python/template

```diff
@@ -1,32 +1,30 @@
 # Template file for 'libxml2-python'
 pkgname=libxml2-python
-version=2.9.9
-revision=3
+version=2.9.10
+revision=1
 wrksrc="${pkgname%-python}-${version}"
 build_wrksrc=python
-build_style=python-module
-hostmakedepends="python-devel python3-devel"
-makedepends="libxml2-devel python-devel python3-devel"
-depends="python python3"
-short_desc="Library providing XML and HTML support - Python2 bindings"
+build_style=python3-module
+hostmakedepends="python3-devel"
+makedepends="libxml2-devel python3-devel"
+depends="python3"
+short_desc="Library providing XML and HTML support"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT"
 homepage="http://www.xmlsoft.org/"
 distfiles="http://xmlsoft.org/sources/libxml2-${version}.tar.gz"
-checksum=94fb70890143e3c6549f265cee93ec064c80a84c42ad0f23e85ee1fd6540a871
+checksum=aafee193ffb8fe0c82d4afef6ef91972cbaf5feea100edc2f262750611b4be1f
 
 pre_build() {
-	sed "s:/usr/include:${XBPS_CROSS_BASE}/usr/include:" -i setup.py
+	sed -i "s:/usr/include:${XBPS_CROSS_BASE}/usr/include:" setup.py
 }
 
 post_install() {
 	vlicense ../COPYING
 }
 
 libxml2-python3_package() {
-	short_desc+="${short_dest/Python2/Python3}"
-	pkg_install() {
-		vmove "usr/lib/python3*"
-		vlicense ../COPYING
-	}
+	depends="libxml2-python"
+	short_desc+=" - transitional dummy package"
```

Comment:
This is also wrong. It breaks the naming convention and replaces one package with another one that doesn't offer the same functionality and thus will break packages depending on it.

I think the best course of actions would be to make libxml2-python3 a separate package with its own template which will have to happen anyway when python2 is dropped. Then issues with the python 2 bindings can be handled separately.

Reply:
which ones depend on python2? I only found `gnome-doc-utils` and bumped it to python3 with a patch.

Reply:
`bamf`, `libGL`, and `libxslt` seem to be the other three

Reply:
Indeed. Removing the python2 version is alright if nothing else depends on it, but not if it is done this way. You should just remove libxml2-python and make libxml2-python3 a separate package.